vendor/knative.dev/hack/infra-library.sh

```diff
@@ -117,29 +117,38 @@ function create_kind_test_cluster() {
 }
 
 # Create a GKE test cluster with kubetest2 and run the test command.
-# Parameters: $1 - custom flags defined in kntest
-#             $2 - test command to run after the cluster is created (optional)
+# Parameters: $1 - custom flags defined in kubetest2
+#             $2 - custom flags to pass directly to gcloud
+#             $3 - test command to run after the cluster is created (optional)
 function create_gke_test_cluster() {
   local -n _custom_flags=$1
-  local -n _test_command=$2
+  local -n _extra_gcloud_flags=$2
+  local -n _test_command=$3
 
   # We are disabling logs and metrics on Boskos Clusters by default as they are not used. Manually set ENABLE_GKE_TELEMETRY to true to enable telemetry
   # and ENABLE_PREEMPTIBLE_NODES to true to create preemptible/spot VMs. VM Preemption is a rare event and shouldn't be distruptive given the fault tolerant nature of our tests.
-  local extra_gcloud_flags=""
   if [[ "${ENABLE_GKE_TELEMETRY:-}" != "true" ]]; then
-    extra_gcloud_flags="${extra_gcloud_flags} --logging=NONE --monitoring=NONE"
+    _extra_gcloud_flags="${_extra_gcloud_flags} --logging=NONE --monitoring=NONE"
   fi
 
   if [[ "${ENABLE_PREEMPTIBLE_NODES:-}" == "true" ]]; then
-    extra_gcloud_flags="${extra_gcloud_flags} --preemptible"
+    _extra_gcloud_flags="${_extra_gcloud_flags} --preemptible"
   fi
+
+  _extra_gcloud_flags+=("--quiet")
   if ! command -v kubetest2 >/dev/null; then
     tmpbin="$(mktemp -d)"
     echo "kubetest2 not found, installing in temp path: ${tmpbin}"
     GOBIN="$tmpbin" go install sigs.k8s.io/kubetest2/...@latest
     export PATH="${tmpbin}:${PATH}"
   fi
-  run_kntest kubetest2 gke "${_custom_flags[@]}" \
-    --test-command="${_test_command[*]}" \
-    --extra-gcloud-flags="${extra_gcloud_flags}"
+  if [[ ! " ${_custom_flags[*]} " =~ "--machine-type=" ]]; then
+      _custom_flags+=("--machine-type=e2-standard-4")
+  fi
+  kubetest2 gke "${_custom_flags[@]}" --rundir-in-artifacts --up --down \
```

Comment:
@dprotaso @evankanderson @cardil @kvmware 

Are we happy with this change? I'm going to apply it upstream but every repo will need to make small adjustments similar to the changes I made here to the `test/*.sh` files. This change works and all the e2e tests are passing. Jobs can pass most kubetest2 args and gcloud args from their specific e2e_tests script.

Tl;dr, we need to deprecate kntest and call kubetest2 directly.

Reply:
I don't have an opinion. Just glad we can move on from kntest. 
For the test/*.sh changes it would be good to sed replace those and open at same time, or at the very least make sure that is only things that need changing. Would suck to find some repo's test scripts that require a lot of work to get going and are unable to test.